docs/source/prometheus.rst

```diff
@@ -26,6 +26,22 @@ dask_scheduler_clients
     Number of clients connected
 dask_scheduler_desired_workers
     Number of workers scheduler needs for task graph
+dask_scheduler_gil_contention
+    0-1 value representing GIL contention on scheduler
+
+    .. note::
+       Requires ``gilknocker`` to be installed, and 
+       ``distributed.admin.system-monitor.gil.enabled``
+       configuration to be set.
+
```

Comment:
```suggestion
```

From what I understand, this metric is currently not implemented.

Reply:
It was implemented in #7624

Reply:
I'm talking about the Prometheus metric specifically; unless I miss something, #7624 does not touch anything related to Prometheus.